test/heex_formatter_test.exs

```diff
@@ -328,4 +328,47 @@ defmodule HeexFormatterTest do
 
     assert_formatter_output(input, expected)
   end
+
+  test "handle live_component format" do
+    input = """
+    <div>
+    <%= live_component(MyAppWeb.Components.SearchBox, id: :search_box, on_select: :user_selected, label: gettext("Search User")) %>
+    </div>
+    """
+
+    expected = """
+    <div>
+      <%= live_component(MyAppWeb.Components.SearchBox,
+        id: :search_box,
+        on_select: :user_selected,
+        label: gettext("Search User")
+      ) %>
+    </div>
+    """
+
+    assert_formatter_output(input, expected)
+  end
+
+  test "handle eex form format" do
+    input = """
+    <div>
+    <%= form_for @changeset, Routes.user_path(@conn, :create), [class: "w-full p-3 rounded-md", phx_change: "on_change"], fn f -> %>
+      <%= text_input f, :name %>
+    <% end %>
+    </div>
+    """
+
+    expected = """
+    <div>
+      <%= form_for @changeset,
+               Routes.user_path(@conn, :create),
+               [class: "w-full p-3 rounded-md", phx_change: "on_change"],
+               fn f -> %>
+        <%= text_input(f, :name) %>
```

Comment:
Does it always adds parentheses or does it add according to `.formatter`? We don't need to handle it now, just to understand if that is case we need to handle 🤔

Reply:
it has some options that may allow us to add parentheses or not, like `locals_without_parens`, so I guess we can add it in .formatter and put in our state then pass it to [Code.format_string!](https://hexdocs.pm/elixir/Code.html#format_string!/1)